src/frontend/src/optimizer/plan_node/stream_topn.rs

```diff
@@ -42,7 +42,14 @@ impl StreamTopN {
         };
         let watermark_columns = FixedBitSet::with_capacity(input.schema().len());
 
-        let base = PlanBase::new_stream_with_core(&core, dist, false, false, watermark_columns);
+        let base = PlanBase::new_stream_with_core(
+            &core,
+            dist,
+            false,
+            false,
+            watermark_columns,
+            MonotonicityMap::new(),
```

Comment:
Are TopN's order columns monotonic?

Reply:
I'm afraid not. Currently I tend to define monotonicity as a property ignoring `Op`s. While you did remind me that TopN can produce watermarks.